config/initializers/008-rack-cors.rb

```diff
@@ -25,15 +25,18 @@ def call(env)
     status, headers, body = @app.call(env)
     headers ||= {}
 
-    Discourse::Cors.apply_headers(cors_origins, env, headers) if cors_origins
+    Discourse::Cors.apply_headers(cors_origins, env, headers)
```

Comment:
what is the context for this change?

Reply:
It will apply cdn headers to the files in the assets path. We can't apply headers to those requests without it.

https://github.com/discourse/discourse/commit/60ff06eef18260533826507cb49f5e68c108f07e#diff-b4a813e6b3c902fe3a912c7e0f941ac17714ccf6c372b649434e65d458897cedL36-R39